core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -410,8 +412,9 @@ open class TileInfo {
             "Friendly Land" -> civInfo != null && isFriendlyTerritory(civInfo)
             else -> {
                 if (terrainFeatures.contains(filter)) return true
-                if (baseTerrainObject.uniques.contains(filter)) return true
-                if (terrainFeatures.isNotEmpty() && getTerrainFeatures().last().uniques.contains(filter)) return true
+                if (filter == "Open terrain" && !isRoughTerrain()) return true
+                if (filter == "Rough terrain" && isRoughTerrain()) return true
```

Comment:
Please just put this in the list above as:
"Open terrain" -> !isRoughTerrain()
"Rough Terrain" -> isRoughTerrain()

Reply:
I am ashamed of myself. This will be fixed!